api/src/pcapi/core/users/models.py

```diff
@@ -769,3 +769,16 @@ class UserProFlags(PcObject, Base, Model):
         nullable=False,
     )
     user: User = orm.relationship(User, foreign_keys=[userId], back_populates="pro_flags", uselist=False)
+
+
+class TrustedDevice(PcObject, Base, Model):
+    __tablename__ = "trusted_device"
+
+    userId: int = sa.Column(sa.BigInteger, sa.ForeignKey("user.id", ondelete="CASCADE"), nullable=False, index=True)
+    user: User = orm.relationship(User, foreign_keys=[userId], backref="trusted_devices")
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** C'est mieux d'utiliser back_populates. 
```suggestion
    user: User = orm.relationship(User, foreign_keys=[userId], back_populates="trusted_devices")
```

Reply:
Pourquoi ? De ce que j'avais compris, avec backref on n'avait pas besoin de préciser trustedDevices dans la table `User`, est-ce qu'on préfère la préciser pour trouver l'information plus facilement ?